index.html

```diff
@@ -9,23 +9,26 @@
 </head>
 <body>
     <main>
-        <section class="container">
-            <h2 class="books__header--title">
-                Awesome Books
-            </h2>
+        <h2 class="books__header--title">
+            All awesome books
+        </h2>
+        <div class="container">
             <template class="books__template">
                 <div class="book">
-                    <p class="book__name">Lorem Ipsum</p>
-                    <p class="book__author">Testeroo Testyy</p>
+                    <div class="book-description">
+                        <p class="book__name">Lorem Ipsum</p> <span>by</span>
+                        <p class="book__author">Testeroo Testyy</p>
+                    </div>
                     <button class="book__remove-btn">Remove</button>
-                    <hr class="book__line">
                 </div>
             </template>
-        </section>
+        </div>
+        <hr class="book__line">
+        <h2 class="book__addnew">Add a New Book</h2>
         <form class="add">
-            <input type="text" class="add__title" id="add__title" name="book_title" placeholder="Title" required>
-            <input type="text" class="add__author" id="add__author" name="book_author" placeholder="Author" required>
-            <input type="submit" class="add__add-btn" value="Add">
+            <input type="text" class="add__title input" id="add__title" name="book_title" placeholder="Title" required>
+            <input type="text" class="add__author input" id="add__author" name="book_author" placeholder="Author" required>
+            <input type="submit" class="add__add-btn input" value="Add">
         </form>
     </main>
     <script src="app.js"></script>
```

Comment:
- [ ] Please user proper 2-space indentation for your HTML in order to keep with the best practices.